README.md

```diff
@@ -20,4 +20,7 @@ func _ready():
 
 func _on_joycon_button_click(name):
 	print(name)
+	
+func _process(delta):
+	joycon._process(delta)
```

Comment:
I'm not sure if this is necessary but without that I am not receiving any signals